core/docker/Dockerfile

```diff
@@ -11,7 +11,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM eclipse-temurin:17.0.4_8-jdk
+FROM eclipse-temurin:17-jdk
```

Comment:
A separate commit for unpinning image version would be helpful if I'm being pedantic.

Reply:
Good point, I’ll add a separate commit. I didn’t know they had logical tags when I did the initial conversion to Temurin.